certify-sunbird-insurance.properties

```diff
@@ -0,0 +1,226 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://demo-registry-credentials-service.sunbird:3000/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential
+
+##Life Insurance Credential Configurations
+
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:registry.dev1.mosip.net:identity-service:3e432fe5-bdab-4717-8eaa-a80d79823e58
```

Comment:
Please check this DID. It should be `api.dev.mosip.net` instead of `registry.dev1.mosip.net`

Reply:
making the change